### PR TITLE
fix(amr): add pwd because password is wrong

### PR DESCRIPTION
**Pull request by adlerhurst**

### CODE REVIEW

File: internal/api/oidc/auth_request_converter.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 
 const (
 	amrPassword     = "password"
```
Comment (fforootd) [inline internal/api/oidc/auth_request_converter.go:21]: should we not comment that this is deprecated?

Comment (adlerhurst) [reply internal/api/oidc/auth_request_converter.go:21]: Yes think so too. I also have check the docs

Comment (livio-a) [reply internal/api/oidc/auth_request_converter.go:21]: It's correct in the docs 😏 https://docs.zitadel.com/docs/apis/openidoauth/claims#standard-claims

File: docs/docs/apis/openidoauth/claims.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Please check below the matrix for an overview where which scope is asserted.
 |:-------------------|:-----------------------------------------|-----------------------------------------------------------------------------------------------|
 | acr                | TBA                                      | TBA                                                                                           |
 | address            | `Teufener Strasse 19, 9000 St. Gallen`   | TBA                                                                                           |
-| amr                | `pwd mfa`                                | Authentication Method References as defined in [RFC8176](https://tools.ietf.org/html/rfc8176) |
+| amr                | `pwd mfa`                                | Authentication Method References as defined in [RFC8176](https://tools.ietf.org/html/rfc8176) <br/> Password value is deprecated, please check pwd |
```
Comment (livio-a) [inline docs/docs/apis/openidoauth/claims.md:46]:
```suggestion
| amr                | `pwd mfa`                                | Authentication Method References as defined in [RFC8176](https://tools.ietf.org/html/rfc8176) <br/> `password` value is deprecated, please check `pwd` |
```

**Review by livio-a — APPROVED** [review]

**Pull request merged**